plugins/spotify/start.sh

```diff
@@ -25,12 +25,19 @@ if [[ -n "$SOUND_SPOTIFY_USERNAME" ]] && [[ -n "$SOUND_SPOTIFY_PASSWORD" ]]; the
     --password "$SOUND_SPOTIFY_PASSWORD"
 fi
 
+# SOUND_SPOTIFY_DISABLE_CACHE: Disable Spotify audio cache
+if [[ ! -z "$SOUND_SPOTIFY_DISABLE_CACHE" ]]; then
```

Comment:
```suggestion
if [[ -z "$SOUND_SPOTIFY_ENABLE_CACHE" ]]; then
```

Reply:
Changes committed and pushed, but are failing the "balena build" check - Trying to view details gets me HTTP 401 errors.

(Tested these changes locally many times.)